packages/contracts-bedrock/test/setup/Setup.sol

```diff
@@ -135,30 +137,24 @@ contract Setup {
     function setUp() public virtual {
         console.log("Setup: L1 setup start!");
 
-        // Optimistically etch, label and allow cheatcodes for the Deploy.s.sol contract
-        DeployUtils.etchLabelAndAllowCheatcodes({ _etchTo: address(deploy), _cname: "Deploy" });
-
-        _isForkTest = vm.envOr("FORK_TEST", false);
-        if (_isForkTest) {
+        if (isForkTest()) {
             vm.createSelectFork(vm.envString("FORK_RPC_URL"), vm.envUint("FORK_BLOCK_NUMBER"));
             require(
                 block.chainid == Chains.Sepolia || block.chainid == Chains.Mainnet,
                 "Setup: ETH_RPC_URL must be set to a production (Sepolia or Mainnet) RPC URL"
             );
-
-            // Overwrite the Deploy.s.sol contract with the ForkLive.s.sol contract
-            DeployUtils.etchLabelAndAllowCheatcodes({ _etchTo: address(deploy), _cname: "ForkLive" });
         }
 
-        // deploy.setUp() will either:
-        // 1. deploy a fresh system or
-        // 2. fork from L1
-        // It will then save the appropriate name/address pairs to disk using artifacts.save()
+        // Etch the contracts used to setup the test environment
+        DeployUtils.etchLabelAndAllowCheatcodes({ _etchTo: address(deploy), _cname: "Deploy" });
+        DeployUtils.etchLabelAndAllowCheatcodes({ _etchTo: address(forkLive), _cname: "ForkLive" });
```

Comment:
we just etch both of these into the state, which is less confusing then overwriting the `deploy` address with the code from `ForkLive`.